### PR TITLE
Feat/5360 add weblate to read me and documentation

### PR DESCRIPTION
**Pull request by gbonaspetti**
#5360

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [X] Manual, quality tests have been done
Tests are not needed for translations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,14 @@ For advanced docker-based usage, look at the full [Tracim Docker documentation](
 
 In order to contribute to the Tracim source code, please read [CONTRIBUTING.md](./CONTRIBUTING.md) file
 
+### Translation
+
+You can help Tracim with improvements or new translations, for this we use the [Weblate](https://hosted.weblate.org/projects/tracim/) platform.
```
Comment (raphj) [inline README.md:39]:
```suggestion
You can help us translate Tracim! If you see missing or wrong translations, head over to [Weblate](https://hosted.weblate.org/projects/tracim/).
```

Should we mention "contribute a new language"?

**Review by raphj — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gbonaspetti) [reply README.md:39]:
```suggestion
You can help us translate Tracim! If you want to contribute with a new language or you see missing or wrong translations, head over to [Weblate](https://hosted.weblate.org/projects/tracim/).
```
Like that @raphj?

Comment (raphj) [reply README.md:39]: Yes, maybe contribute with -> contribute

(to contribute something)

File: README.md [hunk 2]
```diff
@@ -34,6 +34,14 @@ For advanced docker-based usage, look at the full [Tracim Docker documentation](
 
 In order to contribute to the Tracim source code, please read [CONTRIBUTING.md](./CONTRIBUTING.md) file
 
+### Translation
+
+You can help us translate Tracim! If you want to contribute a new language or you see missing or wrong translations, head over to [Weblate](https://hosted.weblate.org/projects/tracim/).
```
Comment (Relf-Algoo) [inline README.md:39]:
```suggestion
You can help us with Tracim's translation! If you want to contribute a new language, if you see missing or wrong translations, head over to [Weblate](https://hosted.weblate.org/projects/tracim/).
```

**Review by Relf-Algoo — CHANGES_REQUESTED** [review]
You can help us translate Tracim! If you want to contribute a new language or you see missing or wrong translations, head over to [Weblate](https://hosted.weblate.org/projects/tracim/).

**Review by Relf-Algoo — APPROVED** [review]

**Pull request merged**